### ISSUE TITLE
convolutional_kernels.cu#L239 net.input should be net.input_gpu

### ISSUE THREAD

**Issue by violet17** [pjreddie/darknet#314]
https://github.com/pjreddie/darknet/blob/16686cec576580489ab3c7c78183e6efeafae780/src/convolutional_kernels.cu#L239

**Comment by pjreddie** [comment]
yup! man, i haven't trained without cudnn in a loooong time, thanks
